lexicons/bsky.app/getBadgeMembers.json

```diff
@@ -0,0 +1,39 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.getBadgeMembers",
+  "type": "query",
+  "parameters": {
+    "uri": {"type": "string", "required": true},
+    "cid": {"type": "string", "required": false},
+    "limit": {"type": "number", "maximum": 100},
+    "before": {"type": "string"}
+  },
+  "output": {
+    "encoding": "application/json",
+    "schema": {
+      "type": "object",
+      "required": ["uri", "members"],
+      "properties": {
+        "uri": {"type": "string"},
+        "cid": {"type": "string"},
+        "members": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": ["did", "name", "acceptedAt"],
```

Comment:
I think this `acceptedAt` is intended to reference what is either `createdAt` or `indexedAt` below.